docs/source/install/deb.rst

```diff
@@ -198,18 +198,32 @@ Check out :doc:`/cli` to learn convinient ways to authenticate via CLI.
 
 Install WebUI and setup SSL termination
 ---------------------------------------
+`NGINX <http://nginx.org/>`_ is used to serve WebUI static files, redirect HTTP to HTTPS,
+provide SSL termination for HTTPS, and reverse-proxy st2auth and st2api API endpoints.
+To set it up: install `st2web` and `nginx`, generate certificates or place your existing
+certificates under ``/etc/ssl/st2``, and configure nginx with StackStorm's supplied site config file.
 
-.. todo:: Detail this section
+  .. code-block:: bash
+
+    # Install st2web and nginx
+    apt-get install st2web nginx
+
+    # Generate self-signed certificate or place your existing certificate under /etc/ssl/st2
+    mkdir -p /etc/ssl/st2
+    openssl req -x509 -newkey rsa:2048 -keyout /etc/ssl/st2/st2.key -out /etc/ssl/st2/st2.crt \
+    -days XXX -nodes -subj "/C=US/ST=California/L=Palo Alto/O=StackStorm/OU=Information \
+    Technology/CN=$(hostname)"
 
-* install nginx
-* generate certificate (instructions, pointer to a script)
-* configure nginx - copy files to site-enabled, loosly explain what we are doing here:
+    # Remove default site, if present
+    rm /etc/nginx/sites-enabled/default
+    # Copy and enable StackStorm's supplied config file
+    cp /opt/{????}/st2.conf /etc/nginx/sites-available/
```

Comment:
@dennybaa @DoriftoShoes nginx configuration file needs to be deployed by st2 package to some location, as discussed here: https://stackstorm.slack.com/archives/stackstorm/p1454528784004805

Need to decide on a final location, make it work, and update the doc here.

The source file is https://github.com/StackStorm/st2/blob/v1.3/conf/nginx/st2.conf on both `master` and `v1.3` branch.